<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>DNA Extraction Protocols</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Akila Wijerathna-Yapa</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="research.html">Research</a>
</li>
<li>
  <a href="publications.html">Publications</a>
</li>
<li>
  <a href="Symposia-and-Workshops.html">Symposia &amp; Workshops</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Teaching Courses &amp; Workshops
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Bayliss-Data-Bazaar.html">Bayliss Data Bazaar</a>
    </li>
    <li>
      <a href="Mass-Spectrometry.html">Mass Spectrometry</a>
    </li>
    <li>
      <a href="Spectroscopy-for-Life-Sceinces.html">Spectroscopy for Life Sceinces</a>
    </li>
    <li>
      <a href="Nuclear-Magnetic-Resonance-(NMR)-Spectroscopy.html">Nuclear Magnetic Resonance (NMR) Spectroscopy</a>
    </li>
    <li>
      <a href="Electron-Paramagnetic-Resonance-(EPR)-Spectroscopy.html">Electron Paramagnetic Resonance (EPR) Spectroscopy</a>
    </li>
    <li>
      <a href="Molecular-Cloning.html">Molecular Cloning</a>
    </li>
    <li>
      <a href="Cell-Culture.html">Cell Culture</a>
    </li>
    <li>
      <a href="Spatial-omics.html">Spatial omics</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Laboratory Protocols
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="DNA-Extraction.html">DNA Extraction</a>
    </li>
    <li>
      <a href="RNA-Extraction.html">RNA Extraction</a>
    </li>
    <li>
      <a href="Protein-Extraction.html">Protein Extraction</a>
    </li>
    <li>
      <a href="Yeast-Transformation.html">Yeast Transformation</a>
    </li>
    <li>
      <a href="Yeast-Two-Hybrid.html">Yeast Two-Hybrid</a>
    </li>
    <li>
      <a href="Homozygous-Transgenic-Selection.html">Homozygous Transgenic Selection</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">DNA Extraction Protocols</h1>

</div>


<p>DNA extraction is a fundamental procedure in molecular biology,
underpinning a vast array of downstream applications including PCR,
sequencing, cloning, and genotyping. An effective DNA extraction
protocol must efficiently lyse cells, protect nucleic acids from
degradation, and purify DNA away from proteins, polysaccharides, and
other contaminants.</p>
<p>This page outlines the logic behind DNA extraction, common reagents
and their biochemical functions, and principles for optimizing protocols
across various sample types.</p>
<hr />
<div id="logic-behind-dna-extraction" class="section level3">
<h3>1. Logic Behind DNA Extraction</h3>
<p>A successful DNA extraction protocol typically involves the following
sequential steps:</p>
<ol style="list-style-type: decimal">
<li><strong>Cell lysis</strong>: Disrupts cell membranes and releases
intracellular components.</li>
<li><strong>DNA stabilization</strong>: Inhibits nucleases that degrade
DNA.</li>
<li><strong>Protein and contaminant removal</strong>: Precipitates or
separates proteins, lipids, and polysaccharides.</li>
<li><strong>DNA precipitation or binding</strong>: Concentrates and
isolates DNA.</li>
<li><strong>DNA wash and elution</strong>: Removes residual contaminants
and recovers purified DNA.</li>
</ol>
<p>These steps can be adapted to suit the biological material (plant,
animal, microbial, or environmental) and intended downstream
application.</p>
<hr />
</div>
<div id="chemicals-used-and-their-functions" class="section level3">
<h3>2. Chemicals Used and Their Functions</h3>
<table>
<colgroup>
<col width="27%" />
<col width="72%" />
</colgroup>
<thead>
<tr class="header">
<th>Reagent</th>
<th>Function</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>Tris-HCl</strong></td>
<td>Buffers pH and maintains a stable environment for DNA integrity</td>
</tr>
<tr class="even">
<td><strong>EDTA</strong></td>
<td>Chelates divalent cations (e.g., Mg²⁺), inhibiting DNases</td>
</tr>
<tr class="odd">
<td><strong>SDS (Sodium dodecyl sulfate)</strong></td>
<td>Disrupts lipid membranes and denatures proteins</td>
</tr>
<tr class="even">
<td><strong>CTAB (cetyltrimethylammonium bromide)</strong></td>
<td>Binds polysaccharides and removes complex carbohydrates (plant
DNA)</td>
</tr>
<tr class="odd">
<td><strong>Proteinase K</strong></td>
<td>Degrades proteins, including histones and nucleases</td>
</tr>
<tr class="even">
<td><strong>Phenol-chloroform</strong></td>
<td>Separates proteins from nucleic acids via phase separation</td>
</tr>
<tr class="odd">
<td><strong>Isopropanol or Ethanol</strong></td>
<td>Precipitates DNA from aqueous solution</td>
</tr>
<tr class="even">
<td><strong>NaCl or KCl</strong></td>
<td>Enhances protein precipitation and stabilizes DNA structure</td>
</tr>
<tr class="odd">
<td><strong>RNase A</strong></td>
<td>Degrades RNA to purify DNA preparations</td>
</tr>
</tbody>
</table>
<hr />
</div>
<div id="optimizing-a-dna-extraction-protocol" class="section level3">
<h3>3. Optimizing a DNA Extraction Protocol</h3>
<p>Optimizing DNA extraction requires balancing purity, yield, and
integrity while minimizing cost and time. Consider the following:</p>
<div id="a-sample-specific-challenges" class="section level4">
<h4>a) Sample-Specific Challenges</h4>
<ul>
<li><strong>Plants:</strong> High levels of polysaccharides and
polyphenols can inhibit downstream reactions. Use CTAB and PVP.</li>
<li><strong>Animal tissues:</strong> Rich in proteins and lipids.
Proteinase K and SDS are essential.</li>
<li><strong>Microbes:</strong> Require mechanical lysis (bead beating)
or lysozyme digestion.</li>
<li><strong>FFPE samples:</strong> Require deparaffinization and
prolonged proteinase digestion.</li>
</ul>
</div>
<div id="b-dna-quality-metrics" class="section level4">
<h4>b) DNA Quality Metrics</h4>
<ul>
<li><strong>A260/A280 ratio (~1.8):</strong> Indicates protein
contamination.</li>
<li><strong>A260/A230 ratio (&gt;2.0):</strong> Indicates salt or phenol
contamination.</li>
<li><strong>Gel electrophoresis:</strong> Assesses integrity and
shearing.</li>
</ul>
</div>
<div id="c-inhibitor-removal" class="section level4">
<h4>c) Inhibitor Removal</h4>
<ul>
<li>Use silica column purification or magnetic beads for clean-up.</li>
<li>Incorporate wash buffers with ethanol and Tris to remove
carryover.</li>
</ul>
</div>
<div id="d-yield-maximization" class="section level4">
<h4>d) Yield Maximization</h4>
<ul>
<li>Minimize sample loss during transfer steps.</li>
<li>Avoid over-drying DNA pellets.</li>
<li>Elute in low-salt buffer (e.g., 10 mM Tris, pH 8.0) or nuclease-free
water.</li>
</ul>
<hr />
</div>
</div>
<div id="a-standard-ctab-based-plant-dna-extraction-protocol-example"
class="section level3">
<h3>4. A Standard CTAB-Based Plant DNA Extraction Protocol
(Example)</h3>
<p><strong>Materials:</strong> - CTAB buffer (2% CTAB, 100 mM Tris-HCl
pH 8, 20 mM EDTA, 1.4 M NaCl) - β-mercaptoethanol (freshly added) -
Chloroform:Isoamyl alcohol (24:1) - Isopropanol - Ethanol (70%)</p>
<p><strong>Steps:</strong> 1. Grind ~100 mg fresh/frozen leaf in liquid
nitrogen. 2. Add 700 µL prewarmed CTAB buffer + 1% β-mercaptoethanol. 3.
Incubate at 65°C for 30 minutes. 4. Add equal volume of
chloroform:isoamyl alcohol. Mix and centrifuge. 5. Transfer aqueous
phase. Add 0.7 volume of cold isopropanol. 6. Incubate at -20°C for 1
hour. Centrifuge and pellet DNA. 7. Wash pellet with 70% ethanol.
Air-dry. 8. Resuspend in TE buffer or nuclease-free water.</p>
<hr />
</div>
<div id="conclusion" class="section level3">
<h3>5. Conclusion</h3>
<p>DNA extraction protocols must be tailored to the sample type and
purpose. Understanding the role of each reagent and the logic of
extraction steps allows researchers to troubleshoot and optimize for
high-quality, inhibitor-free DNA. Advances in automation, magnetic bead
purification, and microfluidics continue to evolve this foundati</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
